### PR TITLE
docs: adding License

### PR DESCRIPTION
**Pull request by jinder1s**
All public github repositories should have a license. This adds the appropriate license for this repository.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,438 @@
+Attribution-NonCommercial-ShareAlike 4.0 International
```
Comment (timmc-edx) [inline LICENSE]: I don't think we want the NC variant -- the existing license information in the footer is by-sa, not by-nc-sa.

Comment (timmc-edx) [reply LICENSE]: Hmm... maybe that's actually backed by the edx/docs.edx.org repo? But I'd be surprised if we wanted to use different licenses for these two.

Comment (jinder1s) [reply LICENSE]: that information comes from edx sphinx theme

Comment (jinder1s) [reply LICENSE]: Good point on np by-sa vs by-nc-sa

Comment (jinder1s) [reply LICENSE]: I'll change these with by-sa

Comment (jinder1s) [reply LICENSE]: Okay, changed to proper license

**Review by timmc-edx — APPROVED** [review]

**Pull request merged**